src/dmd/frontend.d

```diff
@@ -96,12 +96,14 @@ Initializes the global variables of the DMD compiler.
 This needs to be done $(I before) calling any function.
 
 Params:
-    handler = a delegate to configure what to do with diagnostics (other than printing to console or stderr).
+    diagnosticHandler = a delegate to configure what to do with diagnostics (other than printing to console or stderr).
+    fatalErrorHandler = a delegate to configure what to do with fatal errors (other than exit(EXIT_FAILURE))
```

Comment:
I think this would be better worded as 
    a delegate to configure what to do with fatal errors (default is to call `exit(EXIT_FAILURE)`).
ditto for the one above (though that need not be done in this PR).

Reply:
Done